### PR TITLE
Update dependencies; install missing peer dependencies

### PR DESCRIPTION
**Pull request by 36degrees**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,16 +28,20 @@
     ]
   },
   "dependencies": {
+    "acorn": "^6.2.0",
+    "ajv": "^6.10.0",
     "autoprefixer": "^9.5.1",
     "clipboard": "^2.0.4",
     "connect": "^3.6.6",
+    "eslint": "^5.16.0",
```
Comment (nickcolley) [inline package.json]: Is eslint correct here, I'd thought standardjs would have sorted this for us?

Comment (36degrees) [reply package.json]: The specific warnings this is addressing are:

```
npm WARN eslint-config-standard@12.0.0 requires a peer of eslint@>=5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-standard-jsx@6.0.2 requires a peer of eslint@>=5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-es@1.4.0 requires a peer of eslint@>=4.19.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-node@7.0.1 requires a peer of eslint@>=4.19.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-react@7.11.1 requires a peer of eslint@^3.0.0 || ^4.0.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-standard@4.0.0 requires a peer of eslint@>=5.0.0 but none is installed. You must install peer dependencies yourself.
```

These all appear to be dependencies of `standard`:

```
$ npm ls eslint-config-standard eslint-config-standard-jsx eslint-plugin-es eslint-plugin-node eslint-plugin-react eslint-plugin-standard
govuk_design_system@ /Users/ollie/Gds/govuk-design-system
└─┬ standard@12.0.1
  ├── eslint-config-standard@12.0.0
  ├── eslint-config-standard-jsx@6.0.2
  ├─┬ eslint-plugin-node@7.0.1
  │ └── eslint-plugin-es@1.4.0
  ├── eslint-plugin-react@7.11.1
  └── eslint-plugin-standard@4.0.0
```

Does that make sense to you?

Comment (nickcolley) [reply package.json]: It's confusing me because standardJS requires eslint to work, so it should already be installed. :thinking:

Comment (36degrees) [reply package.json]: Looking at the change in package.json, eslint 2.13.1 was installed before. So it looks like standard only requires an older version, but the plugins want something newer?

**Pull request merged**